about.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>About Me-Briyana Haywood</title>
+        <link rel="stylesheet" href="styles/styles.css">
+    </head>
+    <body>
+
+        <section class="logo">
+        <img src=""/>
+        </section>
+        <nav>
+            <a href="index.html">Home</a>
+            <a href="about.html">About Me</a>
+            <a href="portfolio.html">Portfolio</a>
+            <a href="hobby-blog.html">Hobby Blog</a>
+        </nav>
```

Comment:
while it's not imperative, think about making your navigation links an unordered list in the future. It is a common practice that you'll see!